src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -29,8 +29,37 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
     MTRDeviceStateUnreachable = 2
 };
 
+/**
+ * This enum is used to specify the type of logs requested from this device.
+ *
+ * The logs types are : End User Support, Network Diagnostics and Crash logs.
+ */
+typedef NS_ENUM(NSUInteger, MTRDiagnosticLogType) {
+    MTRDiagnosticLogTypeEndUserSupport = 0, // End user support logs are requested
+    MTRDiagnosticLogTypeNetworkDiagnostics = 1, // Network Diagnostics logs are requested
+    MTRDiagnosticLogTypeCrash = 2, // Crash logs are requested
+    MTRDiagnosticLogTypeUnknown = 3
```

Comment:
Does this "unknown" value come from the Matter spec somehow? Otherwise I would make that 0 or -1 so 3 can be used for a real new file type that may be added in the future.

Reply:
I don't see 'unknown' in the Diagnostic Logs Cluster, so maybe we don't even need an unknown value here at all?

Reply:
actually you are right. unknown is not needed for my use case. Removed.